src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Extensions/ISymbolExtensions.cs

```diff
@@ -685,8 +685,8 @@ private static bool VerifyGetEnumerator(IMethodSymbol getEnumerator)
                 return x is
                 {
                     Name: WellKnownMemberNames.MoveNextMethodName,
-                    ReturnType: { SpecialType: SpecialType.System_Boolean },
-                    Parameters: { Length: 0 },
+                    ReturnType.SpecialType: SpecialType.System_Boolean,
+                    Parameters.Length: 0,
```

Comment:
❔ Just wondering, is there a new pattern we would recommend for this, or is this still the cleanest?

Reply:
when list patterns come out, one might prefer `Parameters: []`.  I could see it both ways.